examples/gnrc_border_router/README.md

```diff
@@ -69,18 +85,19 @@ USE_DHCPV6=1 make clean all flash
 ```
 
 ## Usage
-Start the `start_network.sh` script by doing on `dist/tools/ethos`:
+The `start_network.sh` script needed for `ethos` and `slip` is automatically run
+if you type
 
 ```bash
-sudo sh start_network.sh /dev/ttyACMx tap0 2001:db8::/64
+make term
 ```
 
-This will execute the needed commands to setup a `tap` interface
-and configure the BR.
+This will execute the needed commands to setup a `tap` (`ethos`) or `tun` (`slip`)
+interface and configure the BR.
 Notice that this will also configure `2001:db8::/64` as a prefix.
 This prefix should be announced to other motes through the wireless interface.
 
-As said previously, `ethos` allows to send IP packets and shell commands.
+As said previously, `ethos` and `slip` allow to send IP packets and shell commands.
```

Comment:
```suggestion
As said previously, `ethos` and `slipdev` allow to send IP packets and shell commands.
```

As it is the device-specific implementation that allows for the multiplexing, not the SLIP spec.